buildMyCube.sh

```diff
@@ -283,7 +283,7 @@ do
 
         -m|--managment)
 
-            if [ "$2" = "True" ]
+            if [ "$2" = "True" || "$2" = "true" ]
```

Comment:
if [ ${2^^} = "TRUE" ]
Should be even more fool proof, allows True, TRUE, trUe.